lib/bright_serializer/attribute.rb

```diff
@@ -14,12 +14,19 @@ def initialize(key, condition, entity, &block)
       @entity = entity ? Entity::Base.new(entity) : nil
     end
 
-    def serialize(object, params)
+    def serialize(class_instance, object, params)
```

Comment:
I'll be name picky for this one, `class_instance` is a bit to generic

```suggestion
    def serialize(serializer_instance, object, params)
```